policyengine_us/parameters/gov/states/vt/tax/income/credits/renter/rate/share_rent.yaml

```
@@ -0,0 +1,14 @@
+description: Vermont multiplies the renter credit value by the following rate for filers that share rent. 
```

Use the inverse to match the law. Amend the Python logic accordingly.
```suggestion
description: Vermont reduces the renter credit by this fraction for filers who reside with people outside their filing unit.
```